textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslator.java

```diff
@@ -47,6 +47,7 @@ public static GramBooleanQuery translate(String regex, int gramLength)
 		GramBooleanQuery simplifiedDNF = GramBooleanQuery.simplifyDNF(dnf);
 		
 	    TranslatorUtils.escapeSpecialCharacters(simplifiedDNF);
+	    TranslatorUtils.toLowerCase(simplifiedDNF);
```

Comment:
Add a comment to the codebase to explain why we do this.

"Since the inverted index relies on lower-case grams, we need to convert the characters to lower case."